package.json

```diff
@@ -72,9 +72,11 @@
     "@types/leaflet": "^1.5.19",
     "@types/lodash-es": "^4.17.3",
     "@types/qrcode": "^1.3.5",
+    "@types/react": "^17.0.1",
```

Comment:
why did you downgrade @types/react?

Reply:
Maybe because you updated it in the meantime and I was using the old one.

Reply:
I have updated it to 17.0.3 (the newest version)